### DISCUSSION TITLE
Welcome to Windows-10-Activator Discussions!

### DISCUSSION THREAD

**Discussion by chethanyadav456** [chethanyadav456/Windows-10-Activator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Irullinux** [comment]
work in my pc tanks

**Reply by chethanyadav456** [reply]
Good to hear and if have any ideas for this project just let me know.
